packages/cli/src/commands/start.ts

```diff
@@ -200,6 +264,10 @@ export class Start extends Command {
 					);
 				}
 
+				if (!config.getEnv('endpoints.disableUi')) {
+					await Start.generateStaticAssets();
```

Comment:
This still generates 50MB+ of garbage that the GC needs to collect at startup. I'm wondering if we can move this part to a separate process that happens before n8n starts 🤔 .